tests/lowering/tensor_manipulation/test_repeat.py

```diff
@@ -12,10 +13,33 @@ def forward(self, x, sizes):
         return x.repeat(sizes)
 
 
-@pytest.mark.xfail(reason="lowering issue (#67)")
 @pytest.mark.parametrize(
     "input_shape, sizes",
-    [((4, 4), (3, 2))],
+    (
+        ((1, 1, 1), (1, 1, 1)),
+        ((1, 1, 2048, 2048), (1, 1, 1, 1)),
+        ((1, 1, 256), (1, 1, 1)),
+        ((1, 50, 256), (1, 1, 1)),
+        ((100, 1, 256), (1, 1, 1)),
+        ((4, 2), (1, 1)),
+        ((4, 2), (1444, 1)),
+        ((4, 2), (9, 1)),
+        ((6, 2), (1, 1)),
+        ((6, 2), (100, 1)),
+        ((6, 2), (25, 1)),
+        ((6, 2), (361, 1)),
+        ((6, 2), (4, 1)),
+        ((6, 2), (400, 1)),
+        ((6, 2), (9, 1)),
+        pytest.param(
+            (4, 4),
+            (3, 2),
+            marks=pytest.mark.xfail(
+                reason="Current repeat implementation requires aligned last dim when repeating on last dim"
+            ),
+        ),
+        ((5, 16), (2, 3)),
```

Comment:
These cases are not in the list of input variations.  I leave them for future improvement.